.gitignore

```diff
@@ -1,2 +1,3 @@
 frontend/node_modules/
 backend/node_modules/
+frontend/dist/
```

Comment:
in real life it makes sense to have two separate `.gitignore` files for frontend and backend. It makes it easier to manage them. And this global/root `.gitignore ` manages global ignores (envs/infrastructure/db/etc.)

Reply:
Could we keep as it is?

Reply:
As you wish.